RickAndMorty-iOS/Views/Characters/CharacterDetailsView.swift

```diff
@@ -0,0 +1,155 @@
+//
+//  CharacterDetailsView.swift
+//  RickAndMorty-iOS
+//
+//  Created by Gagan on 2023-03-10.
+//
+
+import UIKit
+import SnapKit
+
+class CharacterDetailsView: UIView {
+
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+    }
+
+    init() {
+        super.init(frame: .zero)
+        setupViews()
+        setupConstraints()
+    }
+
+    private func setupViews() {
+        self.backgroundColor = .systemBackground
+        self.addSubview(collectionView)
+        collectionView.accessibilityIdentifier = "CharacterDetailsView"
```

Comment:
This is a so called magic values. Please make separate enum for accessibilityIdentifier values